### PR TITLE
Introduce DHCP controlled management and provision networks for undercloud vm.

### PR DESCRIPTION
**Pull request by aosorgin**
Introduce DHCP controlled management and provision networks for undercloud vm.
Move common variables for create\clean_env scripts.

### CODE REVIEW

File: tripleo/create_env.sh [hunk 1]
```diff
@@ -200,123 +211,6 @@ if ! lsmod |grep '^nbd ' ; then
...
-  local prov_mac=$6
-  local prepare_contrail_pkgs=$7
-
-  # configure eth0 - management
-  _change_iface $mgmt_templ 'eth0' $mgmt_network $mgmt_mac
-
-  # configure eth1 - provisioning
-  _change_iface $prov_templ 'eth1' $prov_network $prov_mac
-
-  # configure root access
```
Comment (alexey-mr) [inline tripleo/create_env.sh]: dont forget to add into patch

File: tripleo/create_env.sh [hunk 2]
```diff
@@ -200,123 +211,6 @@ if ! lsmod |grep '^nbd ' ; then
...
-
-  # prepare contrail pkgs
-  if [[ "$CONTRAIL_SERIES" == 'release' ]] ; then
-    build_series=''
-  else
-    build_series='cb-'
-  fi
-
-  if [[ "$prepare_contrail_pkgs" == 'yes' ]] ; then
-    rm -rf $tmpdir/root/contrail_packages
```
Comment (alexey-mr) [inline tripleo/create_env.sh]: dont forget add it to patching

File: tripleo/create_env.sh [hunk 3]
```diff
@@ -200,123 +211,6 @@ if ! lsmod |grep '^nbd ' ; then
...
-    cp $latest_ver_rpm $tmpdir/root/contrail_packages/
-    # WORKAROUND to bug #1767456
-    # TODO: remove net-snmp after fix bug #1767456
-    mkdir -p $tmpdir/root/contrail_packages/net_snmp
-    cp /home/jenkins/net-snmp/* $tmpdir/root/contrail_packages/net_snmp/
-  fi
-
-  # cp rhel account file
-  if [[ "$ENVIRONMENT_OS" == 'rhel' ]] ; then
-    local rhel_account_file_dir=$(dirname "$RHEL_ACCOUNT_FILE")
```
Comment (alexey-mr) [inline tripleo/create_env.sh]: dont forget add it to patch

File: tripleo/create_env.sh (hunk 1 above)
Comment (aosorgin) [reply tripleo/create_env.sh]: Done

File: tripleo/create_env.sh (hunk 2 above)
Comment (aosorgin) [reply tripleo/create_env.sh]: Done

File: tripleo/create_env.sh (hunk 3 above)
Comment (aosorgin) [reply tripleo/create_env.sh]: Done

**Pull request merged**